packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -388,19 +392,29 @@ class AppBarState extends State<AppBar> {
       );
     }
     if (config.actions != null && config.actions.isNotEmpty) {
+      // Expand to at least 48dp if IconButton or PopupMenuButton.
+      List<Widget> sizedActions = config.actions.map<Widget>((Widget action) {
+        return action is IconButton || action is PopupMenuButton ?
```

Comment:
Ah, got it. We can expand everything. Though if action is a container that has a constraint that has a gesture detector, we might not increase the actual touch target. I guess we can just document that. 

> Also, we tend not to use map because it's less efficient than just writing out the loop.

That's crazy lol. Should we file a bug? Feels like map should be more parallelizable.